packages/gitgraph-react/src/Message.tsx

```diff
@@ -32,7 +27,9 @@ export class Message extends React.Component<MessageProps> {
     const y = commit.style.dot.size;
 
     return (
-      <g ref={ref} transform={`translate(0, ${y})`}>
+      // 0, 14 on buggy
+      // 150, 14 on non-buggy
```

Comment:
🧹  I think these were trails of debugging. I'd get rid of those now the bug is fixed.